bundles/org.openhab.binding.salus/src/main/java/org/openhab/binding/salus/internal/handler/CloudBridgeConfig.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.salus.internal.handler;
+
+import static org.openhab.binding.salus.internal.SalusBindingConstants.SalusCloud.DEFAULT_URL;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * @author Martin Grześlowski - Initial contribution
+ */
+@NonNullByDefault
+public class CloudBridgeConfig {
+    private String username = "";
+    private String password = "";
+    private String url = "";
+    private long refreshInterval = 30;
+    private long propertiesRefreshInterval = 5;
```

Comment:
It is used for building the cache and the API accepts long so there is no reason to limit this value to int

```java
this.devicePropertiesCache = Caffeine.newBuilder().maximumSize(10_000)
        .expireAfterWrite(Duration.ofSeconds(config.getPropertiesRefreshInterval()))
        .refreshAfterWrite(Duration.ofSeconds(config.getPropertiesRefreshInterval()))
        .build(this::loadPropertiesForDevice);
```

Reply:
Long is still overkill, buit yeah, its minor. Regarding the cache, did you consider the available cache machanism in core? It would prevent a dependency.
https://www.openhab.org/javadoc/latest/org/openhab/core/cache/package-summary ?